site/src/sphinx/auth.rst

```diff
@@ -0,0 +1,104 @@
+.. _`Apache Shiro`: https://shiro.apache.org/
+
+.. _auth:
+
+Authentication and Access Control
+=================================
+
+*Central Dogma* provides the features for authentication and access control.
+
+Authentication
+--------------
+
+For authentication, `Apache Shiro`_ is used as its authentication layer and ``conf/shiro.ini`` file is used
+as its security configuration. Please follow the steps below to turn on the security system of *Central Dogma*.
+
+First, enable the security system by configuring ``conf/dogma.json``. Specify login IDs of the administrators there, too::
+
+    {
+        ...
+        "securityEnabled": true,
+        "administrators": [
+            "system-admin", "dogma-admin"
+        ]
+        ...
+    }
+
+Next, configure ``conf/shiro.ini`` based on your authentication system. A local database system of `Apache Shiro`_
+is used here to show you a simple example::
```

Comment:
How about mentioning that a user can also configure Shiro to enable authentication using LDAP or RDBMS as well?

Reply:
Added